data/development/seeds/004-images.js

```diff
@@ -0,0 +1,25 @@
+
```

Comment:
For seeds we should have at least one example of multiple images for one piece of art just for testing purposes

Reply:
We have updated the seeds file to reflect this request!